documentation/docs/examples.md

```diff
@@ -3,7 +3,7 @@ title: Examples
 ---
 
 See the
-[examples folder](https://github.com/cca-io/rescript-material-ui/tree/master/public/rescript-material-ui/examples)
+[examples folder](https://github.com/cca-io/rescript-material-ui/tree/rescript-material-ui/examples)
```

Comment:
Thanks, but this still seems incorrect.

https://github.com/cca-io/rescript-material-ui/tree/master/examples

should be the correct URL.

Reply:
Yikes! not sure how I made that mistake :D

Reply:
Should be correct now. I'm resolving the conversation but let me know if it's not correct!